native/src/renderRules.h

```diff
@@ -472,6 +473,8 @@ class RenderingRulesStorageProperties
 		R_ONEWAY_ARROWS_COLOR = registerRuleInternal(RenderingRuleProperty::createOutputColorProperty("onewayArrowsColor"));
 		R_IGNORE_POLYGON_AS_POINT_AREA = registerRuleInternal(RenderingRuleProperty::createOutputBooleanProperty("ignorePolygonAsPointArea"));
 		R_ADD_POINT = registerRuleInternal(RenderingRuleProperty::createOutputBooleanProperty("addPoint"));
+
+		R_RENDERING_PRIORITY = registerRuleInternal(RenderingRuleProperty::createOutputIntProperty("renderingPriority"));
```

Comment:
naming is not good, it's not rendering priority. It's priority that's being used during density filter.